test/net/sourceforge/kolmafia/request/PlaceRequestTest.java

```diff
@@ -74,7 +79,75 @@ void itShouldSetTheToolbeltAsAFreePullInTTT() {
     assertEquals(
         -1, freePulls.indexOf(toolbelt), "toolbelt should not be in Free Pulls after TTT fades");
   }
-}
 
-// Generated with love by TestMe :) Please report issues and submit feature requests at:
-// http://weirddev.com/forum#!/testme
+  @Nested
+  class Speakeasy {
+    @BeforeEach
+    public void initializeCharPrefs() {
+      KoLCharacter.reset("PlaceRequestTestFakeSotVisit");
+      KoLCharacter.reset(true);
+    }
+
+    @AfterEach
+    public void resetCharAndPrefs() {
+      KoLCharacter.reset("");
```

Comment:
Why any of this?
Usually, the BeforeEach method would reset Preferences only.
We have many many examples of that.